developer-docs-site/docs/community/index.md

```diff
@@ -9,6 +9,14 @@ We welcome your own [contributions](https://github.com/aptos-labs/aptos-core/blo
 
 As always, adhere to the [Aptos Code of Conduct](https://github.com/aptos-labs/aptos-core/blob/main/CODE_OF_CONDUCT.md) when taking part in our ecosystem.
 
+## Why Aptos?
```

Comment:
Have you considered pulling more from here: https://aptosfoundation.org/currents/why-aptos-8-innovations-powering-aptos-network

That way we can kill the link to the Labs medium, not talk at all about the core team, and focus on the tech and the ecosystem.

Reply:
btw why is this here versus say the main page?

Reply:
> btw why is this here versus say the main page?

We have gotten feedback from reviewers our main page is already overwhelming. Leading with another paragraph or two seemed to be going against those wishes.

Reply:
@davidiw , another option is to create a distinct Why Aptos? page in the Start menu, perhaps as the first page above See what's new, that summarizes the features from the governance page.

And still another option is to rename in navigation and expand the white paper introduction to become the Why Aptos? page:
https://aptos.dev/aptos-white-paper/aptos-white-paper-index

It already offers a features list and is among the most popular pages on the site. What do you think? Thanks!

Reply:
After getting guidance on messaging, I have removed references to Core and linked to the recommended Foundation page. If we want to pull that content in still, perhaps that separate page with a merge of the white paper intro would work. @davidiw , please take another look.